src/p4p/server/raw.py

```diff
@@ -239,33 +316,76 @@ def rpc(self, op):
             except AttributeError:
                 op.done(error="RPC not supported")
 
+        def post(self, value, **kws):
+            _log.debug('POST %s %s', self._pv, value)
+            try:
+                self._pv._exec(None, self._real.rpc, self._pv, value, **kws)
+            except AttributeError:
+                pass
+
+        def close(self, pv):
+            _log.debug('CLOSE %s', self._pv)
+            try:
+                self._pv._exec(None, self._real.close, self._pv)
+            except AttributeError:
+                pass
+
     @property
-    def onFirstConnect(self):
+    def on_open(self):
```

Comment:
This is a leftover from an earlier version in which I thought I'd needed to supply decorator names that wouldn't clash with the function names. It does make the names PEP8 compliant so I've left the change in for consideration.